tests/Rule/ContainsUppercaseCharactersTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Tests\Morebec\Validator\Rule;
+
+use InvalidArgumentException;
+use Morebec\Validator\Rule\ContainsUppercaseCharacters;
+use PHPUnit\Framework\TestCase;
+
+class ContainsUppercaseCharactersTest extends TestCase
+{
+    public function testValidate(){
+        $ruleFirst = new ContainsUppercaseCharacters(1,true);
+        $ruleSecond = new ContainsUppercaseCharacters(1,false);
+        $ruleThird= new ContainsUppercaseCharacters(1,false,"Custom Message");
```

Comment:
It would be better to use different values for number of characters in the tests.
The value 1 causes some false positives here.